client/src/pages/Favorites.js

```diff
@@ -1,10 +1,202 @@
+import styled from 'styled-components/macro';
+import { Link } from 'react-router-dom';
+import cat from '../assets/images/cat.svg';
+import dog from '../assets/images/dog.svg';
+import favoriteIcon from '../assets/images/favoriteIcon.png';
+import organizationIcon from '../assets/images/organization.svg';
+import smallAnimals from '../assets/images/smallAnimals.svg';
+import ArrowIcon from '../assets/images/arrowIcon.svg';
+
 import Navigation from '../components/Navigation';
 
-export default function Favorites() {
+export default function Favorites({
+  favorites,
+  organizations,
+  onToggleFavoritesAndFilter,
+}) {
   return (
-    <>
-      <h1>{''}</h1>
+    <FavoritesWrapper>
+      <h1>Your Favorites</h1>
+      {favorites.map((favorite, index) => (
+        <FavoriteCardWrapper key={favorite._id}>
+          <FavoriteCard>
+            <FavoriteProfile src={favorite.picture} alt='favorite profile' />
+            <FavoriteName>{favorite.name.toUpperCase()}</FavoriteName>
+            <FavoriteIcon
+              src={favoriteIcon}
+              alt='favorite Icon'
+              onClick={() => onToggleFavoritesAndFilter(favorite)}
+              style={
+                favorite.isFavorite
+                  ? {
+                      opacity: '100%',
+                      transition: 'all 0.4s ease-in',
+                      transform: 'scale(1.2)',
+                    }
+                  : { opacity: '40%', transition: 'all 0.5s ease.out' }
+              }
+            />
+            <FavoriteInfo>
+              <FavoriteInfoWrapper>
+                {favorite.type === 'cat' ? (
+                  <CatIcon src={cat} alt='cat icon' />
+                ) : null || favorite.type === 'dog' ? (
+                  <DogIcon src={dog} alt='dog icon' />
+                ) : null || favorite.type === 'small_animals' ? (
```

Comment:
please try lines 43 and 45 without ``` null || ``` and see if it still works

Reply:
Works like a charm, thanks ❤️